rest/servers/spring/src/main/java/com/connexta/ingest/rest/spring/IngestApi.java

```diff
@@ -75,6 +76,12 @@
               required = true)
           @RequestHeader(value = "Accept-Version", required = true)
           String acceptVersion,
+      @ApiParam(
+              value =
+                  "The RFC 3339, section 5.6 date-time format string of the last modified time of the file(s) to be ingested. ",
+              required = true)
+          @RequestHeader(value = "Last-Modified", required = true)
+              OffsetDateTime modifiedTime,
```

Comment:
✏️ rename to `lastModified` because that's how it gets generated

Reply:
Or change both to `lastModifiedTime`.

Reply:
What do you mean by both?

Reply:
I vote for `lastModified` :us:

Reply:
will rename to `lastModified`

Reply:
I've been voted off the island. I concede